src/util/useT.js

```diff
@@ -0,0 +1,14 @@
+import { useIntl } from 'react-intl';
+
+/**
+ * Hook to return translation tag function
+ *
+ * Usage example:
+ *   const t = useT();
+ *   return <div>{t`example.key`}</div>
+ */
```

Comment:
I am not sure for such a small hook the example is needed.

Reply:
I would like to let it be: the tagged template string usage could be not so obvious

Reply:
I would prefer to avoid its usage at all. I don't remember cases where such a template string is used at all. Moreover, the name of the variable is `t`. It is not searchable. `translate` would be better in my opinion and thus the template string could be replaced with a regular function call:

```
const translate = use...

`return <div>{translate(example.key)}</div>
```

Reply:
After additional discussion, I see that it makes sense. But I still see the problem with readability when `t` is restructured like here `const { configurations, t } = useRemoteStorageApi();`. For cases like this, I suggest 

`const{​​​​​​​ configurations, translate: t }​​​​​​​​​​​​​​=useRemoteStorageApi();` which adds more context for `t`.

Reply:
removed the hook, using simple function now